presto-sqlserver/src/main/java/com/facebook/presto/plugin/sqlserver/SqlServerClient.java

```diff
@@ -31,4 +36,31 @@ public SqlServerClient(JdbcConnectorId connectorId, BaseJdbcConfig config)
     {
         super(connectorId, config, "\"", new SQLServerDriver());
     }
+
+    @Override
+    public void commitCreateTable(JdbcOutputTableHandle handle)
+    {
+        StringBuilder sql = new StringBuilder()
+                .append("sp_rename ")
```

Comment:
We can't do that because it's done in two different connections, so the transaction's gone (you get an error "The commit transaction request has no corresponding begin transaction).  also in the interim you could have an insert into, which is done separately, but also needs to be a part of the same transaction.